srcpkgs/kicad-footprints/template

```diff
@@ -1,12 +1,16 @@
 # Template file for 'kicad-footprints'
 pkgname=kicad-footprints
-version=5.1.7
+version=5.1.8
 revision=1
 build_style=cmake
 depends="kicad"
 short_desc="Kicad footprint libraries"
 maintainer="Urs Schulz <voidpkgs@ursschulz.de>"
 license="CC-BY-SA-4.0"
 homepage="http://kicad-pcb.org"
-distfiles="https://github.com/kicad/kicad-footprints/archive/${version}.tar.gz>${pkgname}-${version}.tar.gz"
-checksum=2720e009285fbafa44216417c71d438833398ccb4a23d9c6cce86170983817fa
+distfiles="https://gitlab.com/kicad/libraries/${pkgname}/-/archive/${version}/${pkgname}-${version}.tar.gz"
+checksum=31c3676d71139259f1cc67b3a0254ec8a8659d35f2c51a273f5de542c651a819
+
+post_install() {
+	touch $DESTDIR/usr/share/kicad/modules/Connector_Multicomp.pretty/.empty
+}
```

Comment:
```suggestion
post_install() {
	# otherwise KiCad will complain about a missing library when launched
	touch $DESTDIR/usr/share/kicad/modules/Connector_Multicomp.pretty/.empty
}
```

Also, this change is in the wrong commit.